src/RepoAuditor/Plugins/GitHub/Impl/Common.py

```diff
@@ -10,9 +10,25 @@
 
 
 # ----------------------------------------------------------------------
-def CreateIncompleteDataResult() -> Requirement.EvaluateImplResult:
-    """Create an incomplete data result."""
+def CreateIncompleteDataResult(pat_value_exists: bool) -> Requirement.EvaluateImplResult:  # noqa: FBT001

```

Comment:
I'm actually ok with the need to disable the warning where the function is defined. I thought it had to be disabled at each and every call site.